pkg/images/images.go

```diff
@@ -44,6 +45,7 @@ type Resolver struct {
 type ImageOverrideConfig struct {
 	SystemDefaultRegistry string
 	KubeAPIServer         string
+	KubeProxy             string
```

Comment:
nit: this should go after KubeControllerManager (the Kube images are sorted alphabetically)